core/embed/lib/fonts/fonts.h

```diff
@@ -110,10 +119,16 @@
 #define FONT_MAX_HEIGHT_5 FONT_MAX_HEIGHT_4
 #endif
 
+#ifdef TREZOR_FONT_SUB_ENABLE
+#define FONT_MAX_HEIGHT_6 MAX_FONT_H(FONT_SUB_MAX_HEIGHT, FONT_MAX_HEIGHT_5)
+#else
+#define FONT_MAX_HEIGHT_6 FONT_MAX_HEIGHT_5
+#endif
+
 #ifdef TREZOR_FONT_MONO_ENABLE
 #define FONT_MAX_HEIGHT MAX_FONT_H(FONT_MONO_MAX_HEIGHT, FONT_MAX_HEIGHT_5)
```

Comment:
There should be also `FONT_MAX_HEIGHT_6` on this line.

Reply:
nice catch, fixed in b9690a38e3f8585a230f43405acf1b3e3447c6ef